v2/cmd/integration-test/custom-dir.go

```diff
@@ -0,0 +1,34 @@
+package main
+
+import (
+	"os"
+
+	"github.com/projectdiscovery/nuclei/v2/pkg/testutils"
+)
+
+func getCustomConfigsDir() string {
+	temp := os.TempDir()
+	return temp
+}
+
+type customConfigDirTest struct{}
+
+var customConfigDirTestCases = map[string]testutils.TestCase{
+	"dns/cname-fingerprint.yaml": &customConfigDirTest{},
+}
+
+// Execute executes a test case and returns an error if occurred
+func (h *customConfigDirTest) Execute(filePath string) error {
+	defer os.RemoveAll(getCustomConfigsDir())
```

Comment:
This would remove the whole `/tmp`. We need to create a temporary unique directory:
```go
customTempDirectory, err := os.MkdirTemp("", "")
if err != nil {
	return err
}
defer os.RemoveAll(customTempDirectory)
```

Reply:
Done